src/content/about.md

```diff
@@ -0,0 +1,255 @@
+creating a class that extends the [broccoli-plugin](https://github.com/broccolijs/broccoli-plugin) base class, and
+implementing a `build()` method, that performs some work or returns a promise.
+
+```js
+const Plugin = require('broccoli-plugin');
+
+class MyPlugin extends Plugin
+{
+  build() {
+    // A plugin can receive single or multiple inputs nodes/directories
+    // The files are available in this.inputPaths[0]/this.inputPaths[1]...
+    // You can do anything with these files that you can do in Node
+    // The output of your plugin must write to this.outputPath
+  }
+}
+```
+
+A broccoli-plugin has only one purpose, to transform the files from its `this.inputPaths` directories to its
+`this.outputPath` directory when its `build()` function is invoked. Anything you can do in Node, you can do in
+the `build()` method. A plugin can receive one or multiple inputs, and these are available in the `this.inputPaths`
+array in the order they are provided.
+
+`this.inputPaths` contains paths to directories, that are the `outputPath` of previous plugins. Each `inputPath`
+contains files that you can manipulate and write to `this.outputPath`. Broccoli will handle the state of these
+directories and take responsibility for passing them between plugins.
+
+There is a special case where a `string` is passed as an input to a plugin. When parsing your build pipeline, 
+Broccoli will automatically convert a string input into a
+[source plugin](https://github.com/broccolijs/broccoli-source). This plugin basically connects its input directory
+to its output directory, and also allows Broccoli to `watch` and be notified when files within the input
+directory change and trigger a rebuild. You can also manually create an `unwatched` directory from a string by
+using [UnwatchedDir](https://github.com/broccolijs/broccoli-source#new-unwatcheddirdirectorypath-options), 
+changes in this directory will not trigger a rebuild.
+
+### Trees
+
+The build pipeline is series of connected plugins, from one or more input directories to a single target directory.
+Working from the target directory (the final output) back up to the source directories resembles a tree. Think of a
+piece of Broccoli and you should have a mental model of what a build pipeline looks like.
+
+<div style="text-align: center">
+
+![broccoli tree](/assets/logo/logo-small.png)
+
+</div>
+
+Broccoli parses the result of the `module.exports` from the `Brocfile.js`, and traverses up all of the connected
+nodes all the way to the source directories to produce this tree. 
+
+As Broccoli does this, it sets up the filesystem state for each plugin, creating an `outputPath` for each to write
+to. Broccoli normalizes each plugin into what it calls `nodes`, that contain information about the inputs and 
+output of each item in the build pipeline.
+
+You may often encounter the term "tree" when reading plugin READMEs or in tutorials, just remember a tree is a
+connected set of plugins.
+
+## Building
```

Comment:
should we mention that broccoli guarantees to only be building 1 plugin at a time per pipeline.

Reply:
updated